roles/ceph-osd/tasks/main.yml

```diff
@@ -49,7 +49,7 @@
   when: ceph.bcache_ssd_device is defined
 
 - name: start and add the osd service(s) to the init sequence
-  service: name=ceph
+  service: name=ceph-osd-all
```

Comment:
@nirajdp76 @knandya  It seems this task is not necessary, I would like to remove it.